boards/arm/mm_swiftio/doc/index.rst

```diff
@@ -0,0 +1,230 @@
+| P35       | GPIO_B0_13    | D35      | GPIO2_IO13    |          |               |
++-----------+---------------+----------+---------------+----------+---------------+
+| P36       | GPIO_B0_14    | D36      | GPIO2_IO14    |          |               |
++-----------+---------------+----------+---------------+----------+---------------+
+| P37       | GPIO_B0_15    | D37      | GPIO2_IO15    |          |               |
++-----------+---------------+----------+---------------+----------+---------------+
+| P38       | GPIO_B1_11    | D38      | GPIO2_IO27    |          |               |
++-----------+---------------+----------+---------------+----------+---------------+
+| P39       | GPIO_B1_10    | D39      | GPIO2_IO26    |          |               |
++-----------+---------------+----------+---------------+----------+---------------+
+| P40       | GPIO_B1_9     | D40      | GPIO2_IO25    |          |               |
++-----------+---------------+----------+---------------+----------+---------------+
+| P41       | GPIO_B1_8     | D41      | GPIO2_IO24    |          |               |
++-----------+---------------+----------+---------------+----------+---------------+
+| P42       | GPIO_B1_7     | D42      | GPIO2_IO23    |          |               |
++-----------+---------------+----------+---------------+----------+---------------+
+| P43       | GPIO_B1_6     | D43      | GPIO2_IO22    |          |               |
++-----------+---------------+----------+---------------+----------+---------------+
+| P44       | GPIO_B1_5     | D44      | GPIO2_IO21    |          |               |
++-----------+---------------+----------+---------------+----------+---------------+
+| P45       | GPIO_B1_4     | D45      | GPIO2_IO20    |          |               |
++-----------+---------------+----------+---------------+----------+---------------+
+|           | GPIO_AD_B1_07 |          |               |          |               |
++-----------+---------------+----------+---------------+ I2C0     | I2C_3         |
+|           | GPIO_AD_B1_06 |          |               |          |               |
++-----------+---------------+----------+---------------+----------+---------------+
+|           | GPIO_AD_B1_00 |          |               |          |               |
++-----------+---------------+----------+---------------+ I2C1     | I2C_1         |
+|           | GPIO_AD_B1_01 |          |               |          |               |
++-----------+---------------+----------+---------------+----------+---------------+
+
+
+Programming and Flash
+*************************
+
+Build  applications as usual (see :ref:`build_an_application` for more details).
+
+Configuring a Debug Probe
+=========================
+
+This board is configured by default to use the :ref:`opensda-daplink-onboard-debug-probe`,
+however the :ref:`pyocd-debug-host-tools` do not yet support programming the
+external flashes on this board so you must flash the device by "drag-n-drop"
```

Comment:
Change ``drag-and-drop`` to ``copying files.``  (That's what you do in the example commands below.)